docs/scoring.md

```diff
@@ -0,0 +1,62 @@
+# Goal
+The goal of this document is to explain how scoring works in Lighthouse and what to do to improve your Lighthouse scores across the four sections of the report. 
+
+*Note: if you want a **nice spreadsheet** version of this doc to understand weighting and scoring, check out the spreadsheet [here](https://docs.google.com/spreadsheets/d/1dXH-bXX3gxqqpD1f7rp6ImSOhobsT1gn_GQ2fGZp8UU/edit?ts=59fb61d2#gid=0)*
```

Comment:
you could include a little screenshot if you wanted:

![image](https://user-images.githubusercontent.com/39191/32397461-2d20c87a-c0a7-11e7-99d8-61576113a710.png)

Reply:
added

Reply:
added!